tests/draft2019-09/if-then-else.json

```diff
@@ -264,5 +264,69 @@
                 "valid": false
             }
         ]
-    }
+    },
+    {
+        "description": "then: false should fail if condition matches",
+        "schema": {
+            "if": { "const": 1 },
+            "then": false
+        },
+        "tests": [
+            {
+                "description": "matches if → then=false → invalid",
+                "data": 1,
+                "valid": false
+            },
+            {
+                "description": "does not match if → then ignored → valid",
+                "data": 2,
+                "valid": true
+            }
+        ]
+    },
+    {
+        "description": "else: false should fail when if condition does not match",
+        "schema": {
+            "if": {
+                "properties":{
+                    "name":{"type":"string"}
+                },
+                "required": ["name"]
```

Comment:
This subschema is valid for data `2`, because there is no `type` constraint.  Consequently this test case is valid, not invalid.

Please could you use simpler conditions for all these tests? A simple `"const": 1` like in the first test case would be fine.